servers/rendering/renderer_rd/forward_clustered/scene_shader_forward_clustered.cpp

```diff
@@ -307,8 +308,21 @@ void SceneShaderForwardClustered::ShaderData::set_code(const String &p_code) {
 							continue;
 						}
 
+						RD::PipelineDepthStencilState depth_stencil;
+						if (depth_test != DEPTH_TEST_DISABLED) {
```

Comment:
This is already checked above. Since the check above is needed in the ``else`` branch we can remove this one

Reply:
That wouldn't have worked the way the code was currently structured, since in the next if branch this wasn't checked previously.
I changed things a bit, should be fine now.